packages/safety-suite-api/src/api/users.ts

```diff
@@ -52,6 +52,7 @@ export interface User {
   registered: boolean;
   active: boolean;
   permissions: UserPermission[];
+  termsOfServiceVersion: string;
```

Comment:
This should be updated to be `termsOfServiceVersion?: string;` according to the [Scala case class](https://github.com/idelic-inc/login/blob/3bd08f0807a763f68c9a9f4aae02364323afb606/backend/app/com/idelic/login/user/model/User.scala#L17)

I also verified this on http://login-266.idelic.software/

Before setting `termsOfServiceVersion`:
<img width="851" alt="Screen Shot 2021-03-08 at 9 33 15 AM" src="https://user-images.githubusercontent.com/13290954/110335790-3bfa3a80-7ff2-11eb-9b1d-32a6e8b86872.png">

After setting the `termsOfServiceVersion`:
<img width="811" alt="Screen Shot 2021-03-08 at 9 33 35 AM" src="https://user-images.githubusercontent.com/13290954/110335840-47e5fc80-7ff2-11eb-80f6-6ed7860bc7ef.png">

Resetting the `termsOfServiceVersion` to `null` via the DB will return the user object to the initial state.